### ISSUE TITLE
Create and share deploy code for creating supplemental css files

### ISSUE THREAD

**Issue by meetinghouse** [meetinghouse/cms#56]
John would like to make a change to the light theme, and I am contacting you to see what the best way to do this is. 

It should be pretty simple. The very topmost part of the page, above the big banner image, there is a left-justified spot for a logo (in the sample site it currently say "Your Logo Here") and a set of three links right-justified (Home, About Page, Contact Page).

Is it possible to be able to suppress that whole section so that the banner is actually the top item on the page? 

If so, would that be a new, separate theme, or could it be an option within the existing light theme?

Related to this request, is there a custom CSS file for each separate website that can control things such as background color/picture, font size, style, spacing, link background color, that sort of thing? separate from the master?

**Comment by acavanagh** [comment]
I don't think that there is currently a separate css file for each site,
although if that's useful we can build one in to the deploy script.  Is
this something that you want across all the 'light' theme sites, or just a
one-off customization for a particular site?

**Comment by meetinghouse** [comment]
Thanks for your answer. It would be best to have a css file for each site, even if they don't necessarily get changed for each site. Would you please build into the deploy script? And once that is done, wil I need to re-deploy all the sites, or will the change be pushed from the master?

Also, is it feasible to hide/unhide that top part of the home screen? Being able to turn it on or off would be ideal. But if it can only be one way, then off is preferable. Let me know if I'm being unclear on this.

**Comment by acavanagh** [comment]
I'll make a script to add the css file to each site,  and I can also hide
the top part.  Setting a switch to turn the top part of the home screen on
or off is feasible, and won't take that long, but I'm super-busy right now
so if it's not critical I'd like to put it at the bottom of the list for
now and focus on more high value issues and bugs.  The existing script will
have to be updated on each site that you want the custom file for, but it
will just be a matter of pasting in a line.

**Comment by meetinghouse** [comment]
I concur that this is lower priority than the dark theme work. When you do get back to this, two things:
1, The top part of the page can just get hidden, no toggle needed in the user interface.
2. Once a custom css file is created for each site, we want to be sure that the existing, built-in Admin Settings will still be available. I assume it will, but it's worth asking to be sure. Thanks.

**Comment by acavanagh** [comment]
yep, the custom css file will just be supplemental.

**Comment by meetinghouse** [comment]
Hi. I'm hoping for some deploy code so I can add the supplemental css file to all the sites as I go through and update them, so I'm adding a note here as a reminder. But I also notice that the original request for this issue (to suppress the links at the top nav bar) is no longer wanted, so you can ignore that. I'll change the issue name to reflect this.

**Comment by meetinghouse** [comment]
Also, not to be too crazy about putting extra comments in this issue, in the past any code changes that were made in the master would automatically push to all the sites (just code changes, of course, not content). Is that going to be the case at some point again?

**Comment by acavanagh** [comment]
yes, once we get everything switched over to the newer version of the
codebase we'll put it in on automatic.  There's just been too many drastic
changes over the last couple months to trust it to deploy automatically and
not break anything.

**Comment by meetinghouse** [comment]
Andy,
This is done, we think...
JC
